#!/usr/bin/env python

PACKAGE = "jetbot_pro"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("linear_kp", double_t, 0, "linear kp in PID", 3.0, 0, 10)
gen.add("linear_kd", double_t, 0, "linear kd in PID", 4.0, 0, 10)
gen.add("angular_kp", double_t, 0, "kp in PID", 2.0, 0, 10)
gen.add("angular_kd", double_t, 0, "kd in PID", 2.0, 0, 10)
gen.add("laserAngle", int_t, 0, "laserAngle", 120, 10, 180)
gen.add("PriorityAngle", int_t, 0, "Priority follow Angle", 40, 10, 50)
gen.add("distance", double_t, 0, "distance", 1, 0.1, 8)
gen.add("start", bool_t, 0, "Check to start", True)

exit(gen.generate(PACKAGE, "jetbot_pro", "laserTracking"))
